### PR TITLE
Add "Terms of Use"

### PR DESCRIPTION
**Pull request by forevermatt**

### CODE REVIEW

File: src/views/Terms.svelte [hunk 1]
```diff
@@ -1,9 +1,164 @@
...
+<style>
+h2 {
+  margin-top: 36px;
+  margin-bottom: 18px;
+}
+</style>
+
+<h1 class="py-4">WeCarry Terms of Use</h1>
+<p>Effective December 1, 2019</p>
+<p>Thank you for using WeCarry! We&rsquo;re happy to be able to help in your hand-carry coordination needs. Please read this agreement carefully before using WeCarry to ensure you understand the scope of use and limits of liability for the WeCarry service.</p>
```
Comment (wcjr) [inline src/views/Terms.svelte]: if possible, let's favor simpler characters over entity codes

File: src/views/Terms.svelte [hunk 2]
```diff
@@ -1,9 +1,164 @@
-<h1 class="py-4">Terms of use</h1>
+<script>
+import { tick } from 'svelte'
 
+export let params
+
+$: scrollToSection(params.section)
+
+async function scrollToSection(section) {
+
```
Comment (wcjr) [inline src/views/Terms.svelte]:
```suggestion
```

Comment (wcjr) [reply src/views/Terms.svelte]: unless the whitespace was deliberate

Comment (forevermatt) [reply src/views/Terms.svelte]: The whitespace was deliberate (a blank line to make the comment stand out, and spaces to have the indentation match the correct indentation for that code block, to help diffs line up changes better by maintaining correct indentation even on blank lines).

However, we can certainly remove that if you like it better that way.

Comment (wcjr) [reply src/views/Terms.svelte]: nope, that's good then.

**Pull request merged**